services/client/src/components/video/time-to-video.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+import moment from "moment";
+
+const TimeToVideo = ({ startTime }) => {
+  const now = moment();
+  const eventTime = moment.utc(startTime);
+  const duration = moment.duration(eventTime.diff(now));
+
+  return (
+    <React.Fragment>
+      <style jsx>{`
+        span {
+          color: #dae1e7;
+          font-style: italic;
+        }
+      `}</style>
+      <span>
+        created {duration.humanize(true)}
+      </span>
+    </React.Fragment>
+  );
+};
+
+TimeToVideo.propTypes = {
+  startTime: PropTypes.number.isRequired
+};
+
+export default TimeToVideo;
```

Comment:
❓ Should we make this a generic `TimeTo` component and share it between videos and events?

Reply:
:+1 I made a few modifications to `TimeTo`; I took out the `before/after` as videos only have a `created` date, but I'll follow up this PR with a refactored `TimeTo` as we will need this logic elsewhere going forward.

Reply:
@apoclyps tbf, I'm probably jumping the gun - we can always leave it to rule of threes until we have a solid set of use cases.

Reply:
@alistairjcbrown good call; this will likely change again after Monday when the speaker list is planned over lunch